content/docs/dvclive/api-reference/sklearn.md

```diff
@@ -0,0 +1,116 @@
+# Scikit-learn utilities
+
+See [detailed usage examples](/doc/dvclive/ml-frameworks/sklearn).
+
+## sklearn.log_confusion_matrix
+
+Generates a [_Confusion Matrix_](https://en.wikipedia.org/wiki/Confusion_matrix)
+plot (usable by `dvc plots`) .
```

Comment:
💅 If you're already using `_` a lot in the DVCLive ref then it's OK but probably no need for italics in linked text that already stands out due to underscore and blue color. If anything it should be a CSS style

Reply:
> Usable by dvc plots how?

Nvmd I see it's explained in the description. I'd just maybe put it like this instead:

```suggestion
Generates a [confusion matrix](https://en.wikipedia.org/wiki/Confusion_matrix)
(which can be visualized as `dvc plots`) .
```